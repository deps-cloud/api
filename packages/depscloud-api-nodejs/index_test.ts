describe("api", () => {
    test("require", () => {
        const schema = require("./index");

        expect(Object.keys(schema)).toMatchSnapshot();
    });
});
