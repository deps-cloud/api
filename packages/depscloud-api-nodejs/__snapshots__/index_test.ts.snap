// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`api require 1`] = `
Array [
  "v1alpha",
  "v1beta",
]
`;
